examples/reset_remove.rs

```diff
@@ -1,58 +1,48 @@
 extern crate crdts;
 
-use crdts::{CvRDT, CmRDT, Map, Orswot};
+use crdts::{CmRDT, CvRDT, Map, Orswot};
 
 fn main() {
     let mut friend_map: Map<&str, Orswot<&str, u8>, u8> = Map::new();
 
     let read_ctx = friend_map.len(); // we read anything from the map to get a add context
     friend_map.apply(
-        friend_map.update(
-            "bob", read_ctx.derive_add_ctx(1),
-            |set, ctx| set.add("janet", ctx)
-        )
+        friend_map.update("bob", read_ctx.derive_add_ctx(1), |set, ctx| {
+            set.add("janet", ctx)
+        }),
     );
 
     let mut friend_map_on_2nd_device = friend_map.clone();
 
     // the map on the 2nd devices adds 'erik' to `bob`'s friends
-    friend_map_on_2nd_device.apply(
-        friend_map_on_2nd_device.update(
-            "bob",
-            friend_map_on_2nd_device.len().derive_add_ctx(2),
-            |set, c| set.add("erik", c)
-        )
-    );
+    friend_map_on_2nd_device.apply(friend_map_on_2nd_device.update(
+        "bob",
+        friend_map_on_2nd_device.len().derive_add_ctx(2),
+        |set, c| set.add("erik", c),
+    ));
 
     // Meanwhile, on the first device we remove
     // the entire 'bob' entry from the friend map.
-    friend_map.apply(
-        friend_map.rm(
-            "bob",
-            friend_map.get(&"bob").derive_rm_ctx()
-        )
-    );
```

Comment:
The formatting here was intentional  to show the Remove operation needs to be applied for it to take effect, everything on a single line obscures the behavior here a bit